ipc/api/src/evm.rs

```diff
@@ -81,99 +79,48 @@ macro_rules! cross_msg_types {
             }
         }
 
-        impl TryFrom<StorableMsg> for $module::StorableMsg {
+        impl TryFrom<IpcEnvelope> for $module::IpcEnvelope {
             type Error = anyhow::Error;
 
-            fn try_from(value: StorableMsg) -> Result<Self, Self::Error> {
-                let msg_value = fil_to_eth_amount(&value.value)?;
-                let msg_fee = fil_to_eth_amount(&value.fee)?;
+            fn try_from(value: IpcEnvelope) -> Result<Self, Self::Error> {
+                let val = fil_to_eth_amount(&value.value)?;
 
-                log::info!(
-                    "storable message token amount: {:}, converted: {:?}",
-                    value.value.atto().to_string(),
-                    msg_value
-                );
```

Comment:
@cryptoAtwill why remove this log line?

Reply:
The caller should have logged it instead of in the conversion function.